api-specs.json

```diff
@@ -76,7 +76,7 @@
     "dir": "azure-mgmt-compute",
     "source": "specification/compute/resource-manager/readme.md",
     "package": "com.microsoft.azure.management.compute",
-    "args": "--payload-flattening-threshold=1 --tag=package-2018-10-01"
+    "args": "--payload-flattening-threshold=1 --tag=package-java-build-2019"
```

Comment:
@praries880 it maps to almost the same thing. Laurent and I created package java-build-2019 from 2018-10-01, we just changed gallery's version from 2018-06-01 (in package 2018-10-01) to 2019-03-01 (in java-build-2019). This was because the customer was only asking for updates that were contained in gallery and it's time sensitive.